ikenna-portfolio/portfolio/src/main/webapp/index.html

```diff
@@ -47,97 +47,138 @@ <h3>I'm a Googler!!</h3>
+      </form><br>
       <div class="row">
-        <div class="column">
-          <button onclick="changeVSit()">V Sit</button>
-        </div>
-        <div class="column">
-          <button onclick="changeBackLever()">Back Lever</button>
-        </div>
-        <div class="column">
-          <button onclick="changePistolSquat()">Pistol Squat</button>
-        </div>
-        <div class="column">
-          <button onclick="changeFrontLever()">Front Lever</button>
-        </div>
-        <div class="column">
-          <button onclick="changeHSPU()">Freestanding Handstand Push Up</button>
-        </div>
-      </div>
-      <br>
-      <div class="row">
-        <div class="column">
+        <div class="col-sm-6">
           <img src="" id="cal-image1" class="img-responsive margin image-format" alt="">
           <h3 id="goal"></h3>
         </div>
-        <div class="column"> 
+        <div class="col-sm-6"> 
           <img src="" id="cal-image2" class="img-responsive margin image-format" alt="">
           <h3 id="current"></h3>
         </div>
       </div>
     </div>
 
+    <!--Comments Container-->
+    <div id="comments-sect" class="container-fluid bg-5 text-center">
+      <h3 class="margin">Any Questions, Comments, and Concerns? Post Them Below!</h3>
+      <h4 class="margin">We promise to take your posts seriously >:)</h4>
+      <form action="/data" method="POST" class="desc">
+        <div class="comment-wrapper panel-info panel-body">
+          <textarea name="name-box" class="form-control" placeholder="Name (Not Required)" rows="1"></textarea><br>
+          <textarea name="text-box" class="form-control" placeholder="Questions?, Comments., Concerns!" rows="3"></textarea><br>
+          <input type="submit" class="btn btn-info pull-right">
+          <div class="clearfix"></div><hr>
+        </div>
+      </form>
+      <form action="/count" method="POST" class="desc">
+        <div class="comment-wrapper panel-info panel-body">
+          <div class="row">
+            <div class="col-sm-4 pull-left">
+              <div class="pull-left">
+                <label for="count">
+                  <small id="comment-count"></small>
+                </label><br>
+                <button class="btn btn-info">Go!</button>
+                <input type="number" id="count" name="count" min="1" max="10">
+              </div>
+            </div>
+            <div class="col-sm-4">
+              <label for="filter">
+                <small id="filter-label"></small>    
+              </label><br>
+              <button class="btn btn-info"><i class="glyphicon glyphicon-filter"></i></button>
+              <select name="filter" id="filter" class="btn btn-secondary dropdown-toggle">
+                <option value="">--Please choose a filter--</option>  
+                <option value="oldest">Oldest</option>
+                <option value="newest">Newest</option>
+                <option value="name-a">Name A-Z</option>
```

Comment:
Maybe we can rename to name-asc and name-desc to be more clear.

Reply:
done

Reply:
changed to name_a_z and name_z_a instead for easier enum conversion